src/containers/PermissionsApplication.jsx

```diff
@@ -110,6 +113,15 @@ const PermissionsApplication = ({ t }) => {
           </IconButton>
           <NavigationList>
             <PageTitle>{appName.toUpperCase()}</PageTitle>
+            <OpenappButton
+              type={
+                isKonnector(matchingQueryResult.data[0].type)
+                  ? 'konnector'
+                  : 'app'
```

Comment:
let's update isKonnector. We need only:

const getType = type => type === 'is.cozy.konnectors' ? 'konnector' : 'app'

Reply:
why not using slug?